.changeset/pretty-moles-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"victory-core": patch
+---
+
+Fixes #2475: Improve accuracy of text size measurements
```

Comment:
The changeset changelog plugin we use ([@svitejs/changesets-changelog-github-compact](https://github.com/svitejs/changesets-changelog-github-compact/blob/main/packages/changesets-changelog-github-compact/README.md)) is a little bit picky about magic links and stuff. Here's what I'd suggest for final changelog correctly linking: 

```suggestion
Improve accuracy of text size measurements (fixes #2475)
```